Content.Shared/Doors/Components/PryUnpoweredComponent.cs

```diff
@@ -0,0 +1,8 @@
+namespace Content.Shared.Doors.Prying.Components;
+
+///<summary>
+/// Applied to doors that can be pried open without tools while unpowered
```

Comment:
```suggestion
/// Applied to entities that can be pried open without pryingcomponent while unpowered.
```

Except use the <see cref> for pryingcomponent.